src/tests/build.proj

```diff
@@ -581,6 +581,8 @@
       <GroupBuildCmd>$(GroupBuildCmd) "/p:PackageOS=$(PackageOS)"</GroupBuildCmd>
       <GroupBuildCmd>$(GroupBuildCmd) "/p:RuntimeFlavor=$(RuntimeFlavor)"</GroupBuildCmd>
       <GroupBuildCmd>$(GroupBuildCmd) "/p:RuntimeVariant=$(RuntimeVariant)"</GroupBuildCmd>
+      <GroupBuildCmd>$(GroupBuildCmd) "/p:ServerGarbageCollection=$(ServerGarbageCollection)"</GroupBuildCmd>
+      <GroupBuildCmd>$(GroupBuildCmd) "/p:StripSymbols=$(StripSymbols)"</GroupBuildCmd>
```

Comment:
Looks like this makes the dwarf warnings test to fail

Reply:
Something up the stack sets `StripSymbols` and once propagated it breaks the test. 
 `StripSymbols` is a very useful setting if someone needs to debug tests, but we can deal with that later.